internal/publish/database/exposure_test.go

```diff
@@ -756,6 +756,10 @@ func TestIterateExposuresCursor(t *testing.T) {
 	if err == nil {
 		t.Fatal("expected error")
 	}
+	// IterateExposures doesn't guarantee the order, sort to make this test deterministic
+	sort.Slice(seen, func(i, j int) bool {
+		return seen[i].IntervalNumber < seen[j].IntervalNumber
```

Comment:
Do you want to sort by b64 TEK, then IntervalNumber, then rolling period?

Reply:
For this case I think IntervalNumber is sufficient. Also a QQ: how do we compare b64 TEK?

Reply:
It's just a string, so `<` or `>`.

Reply:
I thought they are generated from random hashes, are they guaranteed to be ordered?

Reply:
They are random, but when you're sorting two lists to compare them, you just sort them like strings.

Reply:
Yea